Solutions/PostgreSQL/Data Connectors/Connector_PostgreSQL.json

```diff
@@ -0,0 +1,168 @@
+{
+    "id": "PostgreSQL",
+    "title": "PostgreSQL Events",
+    "publisher": "PostgreSQL",
+    "descriptionMarkdown": "PostgreSQL data connector provides the capability to ingest [PostgreSQL](https://www.postgresql.org/) events into Microsoft Sentinel. Refer to [PostgreSQL documentation](https://www.postgresql.org/docs/current/index.html) for more information.",
+    "additionalRequirementBanner": "This data connector depends on a parser based on Kusto Function to work as expected. Follow the steps to use this Kusto Function alias **PostgreSQLEvent** in queries and workbooks. [Follow steps to get this Kusto Function>](https://aka.ms/sentinel-postgresql-parser)",
```

Comment:
Please validate and provide proper link as it is redirecting to bing https://aka.ms/sentinel-postgresql-parser

Reply:
Such links are handled by the Microsoft team. We just put the proper text into data connector template

Reply:
@spsocprime, @mkchiliveri, handled and its working, can you please check, thanks.